news_sentiment_analyser/pipeline_analysis.py

```diff
@@ -6,9 +6,15 @@
 
 def pipeline() -> None:
     """The full elt pipeline."""
-    articles = extract()
-    articles = transform(articles)
-    load(articles)
+    try:
+        articles = extract()
+        print("Articles extracted!")
+        articles = transform(articles)
+        print("Articles transformed.")
+        load(articles)
+        print("Articles inserted.")
+    except Exception as err:
+        print(f"Error occurred: {err}")
```

Comment:
Good idea to include error handling and print statements, makes it much clearer when things have either worked or failed :)